ha/setup/ha_setup.py

```diff
@@ -397,19 +444,24 @@ def process(self):
             else:
                 # Add node with SSH
                 self._add_node_remotely(node_name, cluster_user, cluster_secret)
+                # configure stonith for each node from that node only
+                configure_stonith(push=True, stonith_config=all_nodes_stonith_config.get(node_name))
```

Comment:
In one case, configure_stonith is called in create_pacemaker_resources and remaining cases it's called from ha_setup. Why can't it be always called from ha_setup?

Reply:
Done.